pkg/controller/config/config_controller_test.go

```diff
@@ -477,6 +478,14 @@ func setupController(ctx context.Context, mgr manager.Manager, wm *watch.Manager
 // Verify the Opa cache is populated based on the config resource.
 func TestConfig_CacheContents(t *testing.T) {
 	ctx, cancelFunc := context.WithCancel(context.Background())
+	once := gosync.Once{}
+	testMgrStopped := func() {
```

Comment:
Seems like all of these could just be:

```
defer cancelFunc
```

Or am I missing something?

Reply:
i think this is vestigial at this point; we can clean it up.